Makefile

```diff
@@ -19,7 +19,7 @@ release:
 
 .PHONY: release-static
 release-static:
-	RUSTFLAGS="-C target-feature=+crt-static" cargo build --release --target x86_64-unknown-linux-gnu
+	RUSTFLAGS="-C target-feature=+crt-static -L /usr/lib/x86_64-linux-gnu" cargo build -v --release --target x86_64-unknown-linux-gnu --features static
```

Comment:
`-L /usr/lib/x86_64-linux-gnu` is not passed down to the `libbpf-sys` crate, which makes it fail to find libelf as well as libz.

Reply:
Running out of ideas, opened https://github.com/libbpf/libbpf-sys/issues/49